src/styles/index.scss

```diff
@@ -1,3 +1,300 @@
+@import 'variables';
+
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
```

Comment:
The `box-sizing: border-box;` property is usually applied to all elements on the page using the universal selector `*`. This ensures a consistent box model is used throughout your styles.